.github/workflows/proto-docker.yml

```diff
@@ -29,18 +28,19 @@ jobs:
           platforms: all
 
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@latest
```

Comment:
can we use versions, this can cause unforeseen issues if a new breaking version gets released